src/main/java/cheese/CheeseException.java

```diff
@@ -0,0 +1,14 @@
+package cheese;
+
+/**
+ * Custom exception
+ */
+public class CheeseException extends Exception {
+    /**
+     * Just a constructor
```

Comment:
This is a very minor comment but I think there should be a blank space in between the descriptor and any @param or @return. Each line should also be concluded with a full stop.